demos/apps/bookinfo/bookinfo-istio.yaml

```diff
@@ -127,12 +127,14 @@ spec:
         ports:
         - containerPort: 9080
       - name: proxy
-        image: docker.io/istio/runtime:20170224214559
+        image: docker.io/istio/runtime:a87bea5
         securityContext:
           runAsUser: 1337
         args:
           - proxy
           - sidecar
+          - -n
```

Comment:
This restricts traffic to a namespace and helps to reduce load on the controller and the restart frequency. We can add another flag to use the current namespace, which I like.